README.md

```diff
@@ -28,7 +28,30 @@ You should be able to get a good idea as to how other people are customising the
 
 ### Offline usage
 
-If you want to use an existing config available in `nf-core/configs`, and you're running on a system that has no internet connection, you'll need to download the config file and place it in a location that is visible to the file system on which you are running the pipeline. You can then run the pipeline with the `-c` parameter - see [Testing](#testing) for example.
+If you want to use an existing config available in `nf-core/configs`, and you're running on a system that has no internet connection, you'll need to download the config file and place it in a location that is visible to the file system on which you are running the pipeline. Then run the pipeline with `--custom_config_base`
+or `params.custom_config_base` set to the location of the directory containing the repository files:
+
+```bash
+## Download and unizp the config files
+cd /path/to/my/configs
+wget https://github.com/nf-core/configs/archive/master.zip
+unzip master.zip
+
+## Run the pipeline
+cd /path/to/my/data
+nextflow run /path/to/pipeline/ --custom_config_base /path/to/my/configs/configs-master/
```

Comment:
This implies that you can run the pipeline from a directory? Most of the pipelines use a single `main.nf` script right?

Reply:
It works with or without the `main.nf`. If you point to a directory that has `nextflow.config` and `manifest.mainScript` etc it works with just the directory.

Reply:
This I didnt know :+1: